.buildkite/pipeline.yml

```diff
@@ -71,5 +111,31 @@ steps:
       platform: x86_64.metal
```

Comment:
Since you are already using v2, can you also add the clippy test for aarch64?
```
  - label: "clippy-arm"
      commands:
       - cargo clippy --all
      retry:
        automatic: false
      agents:
        platform: arm.metal
      plugins:
        - docker#v3.0.1:
            image: "rustvmm/dev:v2"
            always-pull: true
```

I was just doing that for kvm-ioctls as well. Starting with Rust 1.34 clippy is available on aarch64 as well.